# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/DecodedStream.h
# Commit: 3b70213ee976
# Full Hash: 3b70213ee976d2b7d3b0f17876b833149e085ffd
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Make GetClock() non-const on MediaSink, and allow the AudioSink to modify the AudioQueue. r=alwu,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133642
# ==============================================================================

diff -r 6323b526b0ac -r 3b70213ee976 dom/media/mediasink/DecodedStream.h
--- a/dom/media/mediasink/DecodedStream.h	Tue May 24 12:59:10 2022 +0000
+++ b/dom/media/mediasink/DecodedStream.h	Tue May 24 13:09:04 2022 +0000
@@ -44,7 +44,7 @@
 
   RefPtr<EndedPromise> OnEnded(TrackType aType) override;
   media::TimeUnit GetEndTime(TrackType aType) const override;
-  media::TimeUnit GetPosition(TimeStamp* aTimeStamp = nullptr) const override;
+  media::TimeUnit GetPosition(TimeStamp* aTimeStamp = nullptr) override;
   bool HasUnplayedFrames(TrackType aType) const override {
     // TODO: bug 1755026
     return false;